packages/messaging/package.json

```diff
@@ -18,7 +18,6 @@
   "dependencies": {
     "@firebase/messaging-types": "0.1.2",
     "@firebase/util": "0.1.10",
-    "lcov-result-merger": "^2.0.0",
```

Comment:
@jshcrowthe not sure why this is in dependencies but might be needed in devdeps?

Reply:
This actually isn't needed at all in this package. We use it at a higher level. I seems it snuck in, in #482